### PR TITLE
1.x virus scan downloads

### PR DESCRIPTION
**Pull request by djowett-ftw**
Virus scans file downloads if collective.clamav is installed and configured/enabled.

Part of the solution for https://github.com/4teamwork/ftw.file/issues/107

### CODE REVIEW

File: ftw/file/locales/de/LC_MESSAGES/ftw.file.po [hunk 1]
```diff
@@ -93,6 +93,11 @@ msgstr "Die existierende Datei wird beim Upload überschrieben.<br />Drücken Si
 msgid "download"
 msgstr "herunterladen"
 
+#. Default: "Download not possible because the file contains a virus (${name})."
+#: ./ftw/file/fields.py:99
+msgid "download_not_possible"
+msgstr ""
```
Comment (buchi) [inline ftw/file/locales/de/LC_MESSAGES/ftw.file.po:99]: German translation:
Download blockiert. In der Datei wurde die Schadsoftware ${name} gefunden.

File: ftw/file/locales/fr/LC_MESSAGES/ftw.file.po [hunk 2]
```diff
@@ -93,6 +93,11 @@ msgstr "Vous êtes sur le point de remplacer le fichier existant. <br />Appuyez
 msgid "download"
 msgstr "télécharger"
 
+#. Default: "Download not possible because the file contains a virus (${name})."
+#: ./ftw/file/fields.py:99
+msgid "download_not_possible"
+msgstr ""
```
Comment (buchi) [inline ftw/file/locales/fr/LC_MESSAGES/ftw.file.po:99]: French translation:
Téléchargement bloqué. Le logiciel malveillant ${name} a été trouvé dans le fichier.

Please ask Niklaus or Andres if this is good French 😉

File: ftw/file/locales/en/LC_MESSAGES/ftw.file.po [hunk 3]
```diff
@@ -93,6 +93,11 @@ msgstr ""
 msgid "download"
 msgstr ""
 
+#. Default: "Download not possible because the file contains a virus (${name})."
+#: ./ftw/file/fields.py:99
+msgid "download_not_possible"
+msgstr ""
```
Comment (buchi) [inline ftw/file/locales/en/LC_MESSAGES/ftw.file.po:99]: Download blocked. The malware ${name} has been found in the file.

Comment (djowett-ftw) [reply ftw/file/locales/en/LC_MESSAGES/ftw.file.po:99]: ok, done

File: ftw/file/locales/de/LC_MESSAGES/ftw.file.po (hunk 1 above)
Comment (djowett-ftw) [reply ftw/file/locales/de/LC_MESSAGES/ftw.file.po:99]: done

**Pull request merged**